documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -500,7 +500,7 @@ Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rul
 
 **Why the rule is important**: Per the ARM SDK guidelines, the AutoRest is used to generated the language SDKs. All the supported language generator currently handles "application/json" but some SDKs may support other content-types. This rule is in place to warn the swagger writter that not all the SDKs would work with other content-types.
 
-**How to fix the violation**: N/A.
+**How to fix the violation**: Please consult with AutoRest to ensure that the content type desired can be supported.
```

Comment:
I'm ok with this, but I don't think it helps fixing the violation, if the service really supports the other types, then we'd have to live with the warning, otherwise, the other types need to be removed from the spec. Anyway, I wouldn't spend much more time on this one, as I think we'll end up removing the rule, we need to agree to a decision based on the thread we have on this.